vscode/oso/syntaxes/polar.tmLanguage.json

```diff
@@ -80,7 +80,7 @@
     },
     "resource-block": {
       "name": "meta.resource-block",
-      "begin": "(resource|actor)\\s+([a-zA-Z_][a-zA-Z0-9_]*(::[a-zA-Z0-9_]+)*)\\s*\\{",
+      "begin": "(resource|actor)\\s+([a-zA-Z_][a-zA-Z0-9_]*(?:::[a-zA-Z0-9_]+)*)\\s*\\{",
```

Comment:
TODO (not in this PR): split this regex up over multiple rules so that `actor\nUser\n{}` is correctly highlighted since `\\s` doesn't seem to match newlines.

Reply:
Ugh, that sucks. Yeah, it seems like regex patterns can only match one line in tmLanguage.

Reply:
Maybe not too bad to solve (in future PR). Since `resource|actor` would be a sufficient pattern to look for to know when a resource block starts. The rest can be patterns in the captures.

Reply:
Yeah I tried a few hacks like `[\\s\\n]+` and with [the alleged multi-line flag](https://macromates.com/manual/en/regular_expressions#syntax_oniguruma) like `(?m:[\\s\\n]+)` / `(?m:\\s|\\n)+` / `(?m:[\\s\\n])+` but gave up pretty quickly.